estate/models/estate_model.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+
+from odoo import fields, models
+
+
+class EstateModel(models.Model):
+    _name = "estate.property"
+    _description = "Estate description"
+    _property = "property"
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False, default=datetime.datetime.now() + datetime.timedelta(days=90))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        string='Orientation',
+        selection=[('north', 'North'), ('south', 'South'), ('west', 'West'), ('east', 'East')])
+    active = fields.Boolean(default=True)
+    state = fields.Selection(
+        string="Estate status",
+        selection=[('new', 'New'), ('offer received', 'Offer Received'), ('offer accepted', 'Offer Accepted'), ('sold', 'Sold'), ('cancelled', 'Cancelled')],
```

Comment:
```suggestion
        selection=[
            ('new', 'New'), 
            ('offer_received', 'Offer Received'), 
            ('offer_accepted', 'Offer Accepted'), 
            ('sold', 'Sold'), ('cancelled', 'Cancelled'),
        ],
```
1. It's generally a good idea to indent selection values when there is a possibility that the list would be extended in the future.
2. the first part of the tuple is a technical key, it's a bad idea to put spaces inside it them. `offer received -> offer_received`